specifyweb/context/views.py

```diff
@@ -722,19 +722,19 @@ def api_endpoints_all(request):
 @require_http_methods(['GET', 'POST', 'HEAD'])
 @cache_control(max_age=86400, public=True)
 @openapi(schema={
-    "get": {
-        "parameters": {
-            "languages": {
-                "name": "languages",
-                "in": "query",
-                "description": "Comma separate list of languages",
-                "example": "en-us,uk-ua,ru-ru",
-                "required": False,
-                "schema": {
-                    "type": "string",
-                },
+    "parameters": [
```

Comment:
This change should be undone. The "languages" parameter is only for the "get" method, not for the "post" method

Reply:
this is the validation error that is thrown without the change to this schema:``

Reply:
fixed